File: tests/retrieval/fixtures/sample.c
/* Sample C fixture for repo-map tests. */

#include <stdio.h>

int add(int a, int b) {
    return a + b;
}

void hello(void) {
    printf("Hello, world!\n");
}

int main(int argc, char **argv) {
    hello();
    return add(1, 2);
}

File: tests/retrieval/fixtures/sample.py
"""Sample Python fixture for repo-map tests."""


def greet(name):
    """Return a greeting."""
    return f"Hello, {name}!"


class Calculator:
    """Simple calculator."""

    def add(self, a, b):
        return a + b

    def multiply(self, a, b):
        return a * b


def farewell():
    """Return a goodbye."""
    return "Goodbye!"
